store.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Store-desktop</title>
+    <link rel="stylesheet" href="store.css">
+    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+</head>
+<body>
```

Comment:
Your store page design is decent. The positioning issue lingers here as well 
![Screenshot 2023-10-12 at 18 26 09](https://github.com/CodeYourFuture/HTML-CSS-Module-Project/assets/13808341/ee9994fa-8c2f-49b4-8ddc-b3e6cb8b35dd)